static/skin/i18n/qqq.json

```diff
@@ -47,5 +47,7 @@
 	"torrent-download-alt-text": "Hint for the icon of torrent download",
 	"library-opds-feed": "Hint for the library OPDS feed link",
```

Comment:
library-opds-feed must be removed from qqq.json

Reply:
I remember removing it hah. Probably sneaked in while I was pressing Ctrl Z repeatedly. Thanks, removed.